src/ResourceManagement/HDInsight/HDInsight/Customizations/ClusterOperations.Create.Customizations.cs

```diff
@@ -508,6 +508,13 @@ private static IEnumerable<Role> GetRoleCollection(ClusterCreateParameters clust
                 roles.Add(sandboxHeadNode);
                 return roles;
             }
+
+            List<DataDisksGroupProperties> workernodedatadiskgroups = null;
+            if(clusterCreateParameters.WorkerNodeDataDisksGroups != null && clusterCreateParameters.WorkerNodeDataDisksGroups.Count > 0)
+            {
```

Comment:
what's this here for? Just set DataDisksGroups = clusterCreateParameters.WorkerNodeDataDisksGroups